src/frontend/context/AppContext.tsx

```diff
@@ -100,6 +102,10 @@ const [AppContentProvider, useApp] = createUseContext(() => {
     const [modal, settModal] = React.useState<IModal>(initalState);
     const [toasts, settToasts] = useState<{ [toastId: string]: IToast }>({});
 
+    useEffect(() => {
+        Modal.setAppElement('#app');
+    }, []);
+
```

Comment:
Etter prat med Daphne og en del søking på nett og på slack så har jeg funnet ut at vi burde endre måten vi bruker modaler for å støtte skjermleser bedre. Ved å mounte modalene på #app så får app aria-hidden=true når modalen er åpen. Det gjør at skjermleseren kun leser opp innholdet i modalen, og ikke alt bak

![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/d723c22f-4437-4a26-a909-9ed0da80d8e7)
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/5bc7ec71-a3bc-48a3-806e-9ceff1b3c047)

Reply:
Visuelt ser det helt likt ut som før: 
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/bc17da7f-9834-4d0a-9d2f-6cbb2a86a0e5)